src/data_provider/src/sysInfoLinux.cpp

```diff
@@ -355,6 +364,118 @@ nlohmann::json SysInfo::getNetworks() const
     return networks;
 }
 
+std::pair<std::string, std::string> parseProcessInfo(char path[])
+{
+    // Open stat file
+    std::pair<std::string, std::string> processInfo;
+    std::ifstream file(path, std::ios::in | std::ios::binary);
+    std::string statContent;
+
+    // Get stat file content
+    std::getline(file, statContent);
+
+    // Parse PID and process name
+    int openParentesysPos = std::string(statContent).find("(");
+    int closeParentesysPos = std::string(statContent).find(")");
+    int spacePos = std::string(statContent).find(" ");
```

Comment:
It is neccesary to cast `statContent` to `std::string` ??

Reply:
Not needed